docs/adr/complexity-audit.md

```
@@ -0,0 +1,67 @@
+# Complexity Audit – Story 14.1
+
+## Executive Summary
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)